mm2src/coins/rpc_command/get_enabled_coins.rs

```diff
@@ -23,17 +23,17 @@ impl HttpStatusCode for GetEnabledCoinsError {
 #[derive(Deserialize)]
 pub struct GetEnabledCoinsRequest;
 
-#[derive(Serialize)]
+#[derive(Debug, Serialize)]
 pub struct GetEnabledCoinsResponse {
-    coins: Vec<EnabledCoinV2>,
+    pub coins: Vec<EnabledCoinV2>,
 }
 
-#[derive(Serialize)]
+#[derive(Debug, Serialize)]
 pub struct EnabledCoinV2 {
     ticker: String,
 }
 
-pub async fn get_enabled_coins(
+pub async fn get_enabled_coins_v2(
```

Comment:
why renaming? and if you must, the module name should be renamed too

Reply:
we have function with identical name, ide had issues with import suggestion, it couldnt find function from rpc_command crate 
https://github.com/KomodoPlatform/komodo-defi-framework/blob/56dce19819852c0e760595efb0c5e09d4baca568/mm2src/coins/lp_coins.rs#L5391

Reply:
then `get_enabled_coins_rpc` should have been the name since I nothing makes it `v2`

Reply:
its version 2, bcz it is literally newer version of same function from legacy dispatcher.
I can update name if you dont like v2, I just want not to have conflicting names for more convenient development
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2445/commits/6cf7b93ae91e87ace4148e2ec4f9133b13edf695